target/linux/mediatek/base-files/etc/inittab

```diff
@@ -1,3 +1,4 @@
 ::sysinit:/etc/init.d/rcS S boot
 ::shutdown:/etc/init.d/rcS K shutdown
 ::askconsole:/usr/libexec/login.sh
+ttyS0::askfirst:/usr/libexec/login.sh
```

Comment:
if this is required then the commandline is broken and should be updated to point at the real console